src/TabBar.js

```diff
@@ -70,7 +71,7 @@ export default class TabBar<T: Route<*>>
     renderLabel: PropTypes.func,
     renderIndicator: PropTypes.func,
     onTabPress: PropTypes.func,
-    tabStyle: View.propTypes.style,
+    tabStyle: ViewPropTypes.style,
```

Comment:
You are right. I'll go ahead and remove these `PropTypes` then, but here's an alternate approach for your consideration that I tested on RN 0.43.4:

```js
import {
  View,
  ViewPropTypes,
} from 'react-native';
...
    tabStyle: ViewPropTypes ? ViewPropTypes.style : View.propTypes.style,
```

The destructuring import assigns undefined if the requested module does not exist in the specified file. Meanwhile, Flow is crippled on 0.43.4 by [this aforementioned problem](https://github.com/facebook/react-native/commit/c7387fefc85fd15dceadfa291857099ed3e848b8), so it doesn't complain about the misnamed import.